draft-kazuho-httpbis-priority.md

```diff
@@ -302,7 +307,71 @@ extension element (see {{!RFC7540}}, Section 5.5).
 
 # Considerations
 
-## Why use an End-to-End Header Field?
+## The Case for a Collaborative Prioritization Process {#collaboration}
+
+HTTP/2 prioritization places a lot of responsibility on the client. It provides
+a very flexible dependency-priority scheme that has the potential to model rich
+and diverse use cases. However, experience shows that HTTP/2 implementations
+have not used this for much innovation, instead preferring simpler structures.
+
+The client is well placed to decide request priority based on information such
+as the resource type and the context it will be used in. The goal and how
+information is acted on is described lightly in {{?RFC7540}} Section 5.3:
+
+>  Most importantly, priority can be used to select streams for transmitting
+   frames when there is limited capacity for sending.
+
+> Explicitly setting the priority for a stream is input to a
+  prioritization process.  It does not guarantee any particular
+  processing or transmission order for the stream relative to any other
+  stream.  An endpoint cannot force a peer to process concurrent
+  streams in a particular order using priority.  Expressing priority is
+  therefore only a suggestion.
+
+The client is not alone in understanding the best way to prioritize a request. A
+server's prioritization decision is driven by a number of factors such as the
+selected representation ({{?RFC7231}}, Section 3), data locality, resource
+contention, load balancing across multiple clients, application-specific
+annotations etc. When it generates responses it might determine that the
+client's predicted prioritization was incorrect for how the selected
+representation will be used. Rigidly following the client's suggestion could
+lead to a sub-optimal user experience, and so to avoid this, the server can send
+the response at a different effective priority than the client suggestion.
+
+The prioritization process is a collaborative activity but there are no explicit
+means by which a server can present its information. Attempting to infer reasons
+about prioritization from implicit signals is impractical and error prone due to
+the number of factors involved.
```

Comment:
Re "attempting to infer reasons about prioritization", I think we need to be clear that the subject is an intermediary. Because sending the priority response header to the client does not affect how the responses are prioritized.